sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/SharedGalleryImageTests.java

```diff
@@ -340,7 +340,7 @@ private VirtualMachine prepareGeneralizedVmWith2EmptyDataDisks(
                 .withNewVhd(60)
                 .withCaching(CachingTypes.READ_ONLY)
                 .attach()
-                .withSize(VirtualMachineSizeTypes.STANDARD_D5_V2)
+                .withSize(VirtualMachineSizeTypes.fromString("Standard_D2a_v4"))
```

Comment:
Is it possible to put this to a static method in shared class? So it could be easier to modify if needed.

Reply:
I was planning to do that like your suggestion mentioned. The potential change is to set static value inside `ResourceManagerTestBase` and `Utils`(for sample).

For next live run, some tests might fail with clear error message when the current size does not meet disks/nic configurations. The potential change is to use a large size. This part has been completed in track 1.

Based on above, we need to prepare 2-3 static value for vm size both test base and sample utils. We could add that after live tests pass.